arch/x86/core/ia32/crt0.S

```diff
@@ -34,7 +34,7 @@
 #endif
 
 
-#if defined(CONFIG_SSE)
+#if defined(CONFIG_X86_SSE)
```

Comment:
Is CONFIG_X86_SSE supposed to be a superset of all the configs like CONFIG_X86_SSE4A, for example?

Most of the asm code related to SSE applies to all SSE variants and also MMX since they are all using the same set of XMM registers

I don't think this will work if someone just selects CONFIG_X86_SSE

Reply:
Fixed to do `select X86_SSE` so `CONFIG_X86_SSE` is enabled when any SSE kconfig is enabled.

MMX registers overlay with FPU registers so only needs `CONFIG_FPU`.